<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupEditorDialog</class>
 <widget class="QDialog" name="GroupEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="NotificationFrame" name="notifyFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="GroupEditor" name="groupEditor"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupEditor</class>
   <extends>QTabWidget</extends>
   <header>ui/partial/groupeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotificationFrame</class>
   <extends>QFrame</extends>
   <header>ui/partial/notificationframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>GroupEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GroupEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
